functions/common.ts

```diff
@@ -9,9 +9,11 @@ import type { Issues } from "../.prisma/client";
 
 export interface Assignee {
   name: string;
+  [key: string]: any;
 }
 
 export interface IUnifiedIssue extends Issues {
+  projectID: string;
```

Comment:
Sorry, me and Linus forgot to tell you (and/or add to the readme), but to get the IUnifiedIssue type you now must run `npm run migrate` while standing in the functions folder, then you will see that it still has projectID